docker/nginx/conf.d/default.conf

```diff
@@ -1,12 +1,13 @@
 server {
-    root /srv/api-platform/web;
+    root /srv/app/web;
+    resolver 127.0.0.11;
```

Comment:
I have had friends try this out and it seems it may be an issue on Mac Sierra? Versions before this (e.g. El Capitan) it seems to work fine. Sorry, I can't offer more details at this time, but just FYI.